programmatic_execution_of_notebooks/DemoQCResultsWithParameters.ipynb

```diff
@@ -0,0 +1,1240 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use Papermill Parameters to Customize sample-level QC results\n",
```

Comment:
This notebook should stay in https://github.com/verilylifesciences/variant-qc/blob/main/notebooks/ExploreSampleQCResults.ipynb and just receive these nice updates. I do see the value in having a test notebook along side these two approaches, but I think more of a 'hello world' notebook will do the trick as long as it emits a CSV or a PNG.

We can have two (or more) notebooks in the Terra featured workspace and they do not need to all come from the same git repo.